sql/core/src/test/scala/org/apache/spark/sql/DateFunctionsSuite.scala

```diff
@@ -475,6 +1164,45 @@ class DateFunctionsSuite extends QueryTest with SharedSQLContext {
       Row(ts1.getTime / 1000L), Row(ts2.getTime / 1000L)))
   }
 
+  test("to_unix_timestamp with session local timezone") {
```

Comment:
I agree that there are so many similar tests, but I have no idea to generalize them.
Would you please give me some code snippets? I'll be able to expand them.